website/docs/reference/resource-properties/unit-tests.md

```diff
@@ -0,0 +1,198 @@
+---
+title: "About unit tests property"
+sidebar_label: "Unit tests"
+resource_types: [models]
+datatype: test
+---
+
+## Definition
+
+Unit tests validate your SQL modeling logic on a small set of static inputs before you materialize your full model in production. They support a test-driven development approach, improving both the efficiency of developers and code reliability.
+
+To run only your unit tests, use the command:
+`dbt test --select test_type:unit`
+
+<file name='dbt_project.yml'>
+
+```yml
+
+unit_tests:
+  - name: <test-name> # this is the unique name of the test
+    model: <model-name> 
+      versions: #optional
+	include: <list-of-versions-to-include> #optional
```

Comment:
Not sure what this red block is

Reply:
I can't get rid of it no matter how I alter the spacing. I'm gonna write this off as a glitch until I can get into the preview to confirm

Reply:
I used the GitHub UI to add spaces and replace it. I've seen it before but I'll have to Google how to prevent it from happening.